<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifyLogin</class>
 <widget class="QDialog" name="modifyLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modify Login Information</string>
  </property>
  <widget class="QLabel" name="labelOldPw">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Original Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNewAcc">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNewPw">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="oldpw">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="newacc">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="newpw">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="queren">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Modify</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quxiao">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
